server/channels/testlib/helper.go

```diff
@@ -61,6 +61,7 @@ func NewMainHelperWithOptions(options *HelperOptions) *MainHelper {
 	// Unset environment variables commonly set for development that interfere with tests.
 	os.Unsetenv("MM_SERVICESETTINGS_SITEURL")
 	os.Unsetenv("MM_SERVICESETTINGS_LISTENADDRESS")
+	os.Unsetenv("MM_SERVICESETTINGS_CONNECTIONSECURITY")
```

Comment:
This isn't part of this feature, but it caused me a bunch of trouble when running API tests since I have these configured via environment variables so that I don't lose them when recreating my config.json